packages/cli/src/commands/dev.ts

```diff
@@ -34,9 +42,20 @@ export type DevCommandOptions = {
   route: string;
 };
 
+const anonymousIdLocalState = config.getValue('anonymousId');
+const anonymousId = anonymousIdLocalState || uuidv4();
```

Comment:
This UUID is not correlated to anything else in the system. It would only demonstrate if there is a level of stickiness in a developer using the `dev` command. 

However, I still find that metric weak. I'd wait till the next iteration and do this with the actual user id.